tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -92,20 +92,6 @@ export const createGroup = async ({
   return group
 }
 
-const getGroupId = async ({ group, admin }: { group: Group; admin: User }): Promise<string> => {
```

Comment:
whu u remove this api function?

Reply:
This function simply returns the group uuid by hitting the API. But the group uuid was already stored during group creation. So, replacing it with the stored value on `getCreatedGroup` makes this function unused and tends to be removed. 
Is it okay to remove it? 
or may it be needed in the future, so I have not removed it?
or is there any use cases of this function which I have not found?